--- libxfce4util-4.12.1/libxfce4util/xfce-resource.c	2013-04-16 00:41:26.000000000 +0530
+++ xfce-resource.c	2018-09-09 12:37:47.010824782 +0530
@@ -240,7 +240,7 @@
     }
 
   /* XDG fallback */
-  _list[XFCE_RESOURCE_ICONS] = g_slist_append (_list[XFCE_RESOURCE_ICONS], "/usr/share/pixmaps");
+  _list[XFCE_RESOURCE_ICONS] = g_slist_append (_list[XFCE_RESOURCE_ICONS], "@TERMUX_PREFIX@/share/pixmaps");
 
   /* fallback for system which that don't install everything in /usr */
   _list[XFCE_RESOURCE_ICONS] = g_slist_append (_list[XFCE_RESOURCE_ICONS], "/usr/local/share/pixmaps");
